crtm-config.cmake.in

```diff
@@ -4,4 +4,4 @@ include(CMakeFindDependencyMacro)
 # Include dependencies here, e.g.,
 find_dependency(NetCDF REQUIRED Fortran)
 
-include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-targets.cmake")
+#include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-targets.cmake")
```

Comment:
Do you want to remove that line?

Reply:
@climbfuji yep, builds inside and outside of bundles (i.e., stand-alone).   One issue I'm chasing at the moment is getting standalone GSI to pick it up the same way it did with prior CRTM versions.  That will probably end up being a second PR later.  I'll remove the commented line.

Reply:
That sound great. Let me try building the latest commit in spack, too.